src/TLuaInterpreter.cpp

```diff
@@ -3265,202 +3265,322 @@ int TLuaInterpreter::setBackgroundImage( lua_State *L )
 
 int TLuaInterpreter::setLabelClickCallback( lua_State *L )
 {
-    string luaSendText="";
-    if( ! lua_isstring( L, 1 ) )
-    {
-        lua_pushstring( L, "setLabelClickCallback: wrong argument type" );
+    Host * pHost = TLuaInterpreter::luaInterpreterMap[L];
+    if(! pHost) {
+        lua_pushstring( L, tr( "setLabelClickCallback: NULL Host pointer - something is wrong!" )
+                        .toUtf8().constData() );
         lua_error( L );
         return 1;
     }
-    else
-    {
-        luaSendText = lua_tostring( L, 1 );
-    }
-    string luaName="";
-    if( ! lua_isstring( L, 2 ) )
-    {
-        lua_pushstring( L, "setLabelClickCallback: wrong argument type" );
+
+    QString labelName;
+    if( ! lua_isstring( L, 1 ) ) {
+        lua_pushstring( L, tr( "setLabelClickCallback: bad argument #1 type (label name as string expected, got %1!)" )
```

Comment:
Sadly I think that users could - initially be seeing mixed language message at the same time, for example I turned on showing Lua error messages to the main console and tried to get some errors:

![mudlet_snapshot3](https://cloud.githubusercontent.com/assets/6163092/25095288/0755ed3e-2393-11e7-8995-87e186041982.png)

As I see it, in the above the various strings "centerview: ..." would be in the _user's_ language; actually so could the "[ XXXX ]" ones I think.

To be fair the function names are going to stick to being English with the current consensus of opinion and the Lua ones, like the fragment that says:

 .... ":1: unexpected symbol near 
'\<eof>'>

will be in English as that comes from Lua and not from Mudlet - however from the user (not developer) point of view do they want to see __everything__ in English or is it reasonable to at least give __Mudlet__ related problems (which are going to be related to OUR project, and likely only to this project) in their language even if _general/common_ Lua issues (which many more groups of people, using many languages around the :earth_asia: could/can have) get reported in English only?